src/sentry/api/endpoints/integrations/sentry_apps/installation/index.py

```diff
@@ -52,7 +52,9 @@ def post(self, request: Request, organization) -> Response:
         # only published or owned apps are allowed to be installed
         app = SentryApp.objects.filter(slug=slug).first()
         if app is None or (
-            app.status != SentryAppStatus.PUBLISHED and app.owner_id != organization.id
+            app.status != SentryAppStatus.PUBLISHED
+            and app.owner_id != organization.id
+            and not request.user.is_superuser
```

Comment:
```suggestion
            and not superuser_has_permission(request)
```

could we use this helper function? i'm tightening access around superusers in the form of creating a superuser read and a superuser write. only write superusers will be able to hit POST / PUT / DELETE. we should also check for active superuser anyway and that's one of the checks in this function